Odata-docs/support/support-policy.md

```diff
@@ -0,0 +1,127 @@
+---
+title:  "OData libraries support policy"
+description: Support policy of the official OData libraries by Microsoft.
+date:   2024-01-23
+ms.date: 1/23/2023
+author: habbes
+ms.author: clhabins
+---
+
+# Support policy for official .NET OData libraries
+
+**Applies To**:[!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v8.md)][!INCLUDE[appliesto-webapi](../includes/appliesto-webapi-v7.md)][!INCLUDE[appliesto-odatalib](../includes/appliesto-odatalib-v7.md)]
+
+This guide documents official support policy for official OData libraries for .NET distributed by Microsoft.
+
+Supported releases are patched for security and other critical bugs and may receive new features. Only the latest minor and patch of a given release is supported. For example, for Microsoft.OData.Core 7 release, only the least 7.x.y release is supported. Customers are encouraged to always use the latest patch of a given release. In case issues are encountered, the first step should generally be to update to the latest version of that major release.
```

Comment:
what does the least 7.x.y release mean? Should it be the latest 7.x.y?

Reply:
Yes, latest. Thanks.